scripts/spack/packages/camp/package.py

```diff
@@ -3,7 +3,17 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
+import glob
```

Comment:
Just to make sure nothing in here was added by you, @adayton1 ? It is all from an updated spack package, right?

Reply:
Yes, that is right.